modules/io/common/src/main/java/org/locationtech/jts/io/twkb/Varint.java

```diff
@@ -0,0 +1,213 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * 
+ * 
+ * Original file: https://svn.apache.org/repos/asf/mahout/branches/mahout-0.8/core/src/main/java/org/apache/mahout/math/Varint.java
```

Comment:
If we are making use of this file as provided, do we wish to also include the test case [VarintTest.java](https://svn.apache.org/repos/asf/mahout/branches/mahout-0.8/core/src/test/java/org/apache/mahout/math/VarintTest.java)?

This header looks fine and amounts to a dependency on Apache mahout version 0.8 (depending on source code rather than binary)

Reply:
The VarintTest has already been adapted by @groldan : https://github.com/locationtech/jts/pull/854/files#diff-9f94911887b807419c51762589f89bc8ecfdc48eb0b24c01d5919444c3c6acf2

IMO it's useful to have tests that verify behaviour of this utility class, but I'm not opposed to removing it, since it's unlikely that this code will evolve in jts.